src/interface/PremiumLoginPanel/PremiumLoginPanel.js

```diff
@@ -74,21 +75,25 @@ export class PremiumLoginPanel extends React.PureComponent {
           <div className="text">
             <h1>
               <Textfit mode="single">
-                Hello {user.name}.
+                <Trans>Hello</Trans> {user.name}.
```

Comment:
I think this should encapsulate the whole sentence so you can change the order in some languages, needs an ID as well

Reply:
Ah that makes sense, thank you